server/phpunit.xml.dist

```diff
@@ -26,11 +26,14 @@
     <testsuites>
         <testsuite name="Domain">
             <directory>tests/Unit/Administration/Domain</directory>
+            <directory>tests/Unit/Company/Domain</directory>
             <directory>tests/Unit/Core/Domain</directory>
+            <directory>tests/Unit/User/Domain</directory>
```

Comment:
```suggestion
            <directory>tests/Unit</directory>
```

Reply:
Why not only `<directory>tests/Unit</directory>`?

Reply:
In order to simplify and avoid always having to add a new line when creating a domain